app/src/main/java/org/schulcloud/mobile/data/model/RealmString.java

```diff
@@ -0,0 +1,27 @@
+package org.schulcloud.mobile.data.model;
+
+import io.realm.RealmModel;
+import io.realm.annotations.RealmClass;
+
+@RealmClass
+public class RealmString implements RealmModel
```

Comment:
Realm doesn't support arrays of primitives yet, see the [reference](https://realm.io/docs/java/latest/#primitive-lists). There they have linked a gist with a workaround, which I used: https://gist.github.com/cmelchior/1a97377df0c49cd4fca9